ast/traversal.go

```diff
@@ -0,0 +1,14 @@
+package ast
+
+// searchCallback is the kind of function passed to DepthFirstSearch to get
+// notified about the traversed nodes in a gyp ast tree.
+type searchCallback func(expr Node)
+
+// DepthFirstSearch performs a depth-first traversal of expr's syntax tree and
+// invokes callback with the given cbParam for every node found.
```

Comment:
The documentation is now outdated :smile:

Reply:
Ops, thank you!